### ISSUE TITLE
Could you please add parameter [callback, resetPaging] when reload data from Ajax source

### ISSUE THREAD

**Issue by mixedmuffin** [l-lin/angular-datatables#273]
Dear @l-lin,

In datatable API [ajax.reload()](https://datatables.net/reference/api/ajax.reload()), it can set param 'resetPaging' to false then datatable will hold the current paging position. But [DTInstances.reloadData()](http://l-lin.github.io/angular-datatables/#/dataReloadWithAjax) doesn't have this feature. 
Currently, I use `$('#tableName').DataTable().ajax.reload(null, false);` and it work fine. But would be great if you support this feature in DTInstances.reloadData()

**Comment by l-lin** [comment]
Fixed in the upcoming version 0.4.3 (see ##266).

**Comment by mixedmuffin** [comment]
Hi again,

I see your fix in [angular-datatables.renderer.js](https://github.com/l-lin/angular-datatables/blob/ac18c849c9d9754e245bcc8f4c5133fdb19e5c51/src/angular-datatables.renderer.js) at line 396: `_oTable.ajax.reload(null, false);`. It may force the caller to hold the current datatable state.
It would be great if `DTInstances.reloadData()` have parameter same as  [ajax.reload()](https://datatables.net/reference/api/ajax.reload()). Something like this:
```
function reloadData(callback, resetPaging) {
    if (_oTable) {
        _oTable.ajax.reload(callback, resetPaging || true);
    }
}
```
